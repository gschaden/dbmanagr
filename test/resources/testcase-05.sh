db-navigator.py -s skiline@bmw
