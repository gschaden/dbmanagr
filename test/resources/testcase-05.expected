Id	Title	Subtitle	Autocomplete	
8422e1be-16c9-34b7-ab23-e3d6ea0f7ea1	skiline@bmwqa.skiline.net/	Connection	skiline@bmwqa.skiline.net/

